README.md

```diff
@@ -103,6 +104,7 @@ More examples of the python-webapp's usage can be found in this
 * `config_destination` is calculated to be `${ path }/settings.py`
 * `requirements_file` is calculated to be `setup.py`
 * `config_template` is calculated to be `settings.py.erb`
+* `wsgi_module` is calculated to be `# {project_name }.wsgi:application`
```

Comment:
You're inconsistent with this compared to others above, like `config_destination`